test/rhsmlib/test_file_monitor.py

```diff
@@ -35,6 +39,11 @@ def setUp(self):
         self.fsw1 = file_monitor.FilesystemWatcher(self.dir_list)
         self.fsw2 = file_monitor.InotifyFilesystemWatcher(self.dir_list)
 
+        now: float = time.time()
+        self.future = now + 2.0
+        for file in (self.testpath1, self.testpath2, self.testpath3):
+            os.utime(file, (now, now))
```

Comment:
hm do we need to change the times of the test files? they were created few lines before this, so they should be already "up to date"